templates/minimal/README.md

```diff
@@ -37,11 +37,17 @@
 - 🤏 This template is a minimal (in terms of complexity and the number of components)
 template for building a blockchain node.
 
+
 - 🔧 Its runtime is configured with a single custom pallet as a starting point, and a handful of ready-made pallets
 such as a [Balances pallet](https://paritytech.github.io/polkadot-sdk/master/pallet_balances/index.html).
 
+
 - 👤 The template has no consensus configured - it is best for experimenting with a single node network.
 
+
+- For the most up-to-date chain specification, refer to [staging-chain-spec-builder](https://crates.io/crates/staging-chain-spec-builder)
```

Comment:
I would remove this note from the intro. It isn't very relevant from an introduction standpoint for the template, and the latest info for the chain spec isn't necessarily part of `staging-chain-spec-builder`. I would keep the `#Intro` section as before.

You don't need to add a mention about `staging-chain-spec-builder` in other place either. We already have it mentioned in the steps below (e.g. under `Starting a Minimal Template Chain`/`Omni Node`/`Install staging-chain-spec-builder`.